pyproject.toml

```diff
@@ -14,6 +14,7 @@ version_files = ["pyproject.toml:^version"]
 python = ">=3.9,<3.12"
 dbt-core = "1.6.0"
 requests = "^2.31.0"
+google-cloud-storage = "2.13.0"
```

Comment:
I suspect adding this dependency will require the poetry lock file to be rebuilt.

Please run `poetry lock --no-update` and add the updated `poetry.lock` to the PR